### PR TITLE
Allow video thumbnail scaling

### PR DESCRIPTION
**Pull request by bh412**

### CODE REVIEW

File: src/scss/common/elements.scss [hunk 1]
```diff
@@ -16,6 +16,24 @@
   text-decoration: none;
 }
 
+.content-video-container {
+  position: relative;
+  width: 80%;
+  margin-left: 10%;
+  margin-right: 10%;
```
Comment (mlt47) [inline src/scss/common/elements.scss:23]: This affects the aspect ratio, is there a reason it is not width 100%?

File: src/app/components/content/IsaacVideo.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export function IsaacVideo(props: IsaacVideoProps) {
     return <div>
         <div className="no-print content-value text-center">
             { embedSrc ?
-                <iframe ref={videoRef} className="mw-100" title={altText} width="614" height="390" src={embedSrc} frameBorder="0" allowFullScreen/>
+                <div className="content-video-container"><iframe ref={videoRef} className="mw-100" title={altText} width="614" height="390" src={embedSrc} frameBorder="0" allowFullScreen/></div>
```
Comment (mlt47) [inline src/app/components/content/IsaacVideo.tsx:106]: We need to remove width and height attributes to allow the CSS to maintain the aspect ratio for us.

File: src/scss/common/elements.scss (hunk 1 above)
Comment (bh412) [reply src/scss/common/elements.scss]: I thought it looked better for side by side videos, but removing the height/width from the video iframe corrects this

Comment (mlt47) [reply src/scss/common/elements.scss]: I see, that makes sense. Cool 🙂

**Review by mlt47 — APPROVED** [review]

**Pull request merged**